lib/archethic/reward.ex

```diff
@@ -55,8 +55,9 @@ defmodule Archethic.Reward do
     ...>  }
     ...> } = Reward.new_rewards_mint(2_000_000_000)
```

Comment:
You just forgot to update the doctest function which is crashing tests

Reply:
ok